fre/pp/ppyaml_test/am5-yamls.yaml

```diff
@@ -0,0 +1,12 @@
+## FRE properties
+projectID: "am5"
+FRE_VERSION: &fre_version "canopy"
+AM5_VERSION: &am5_version "2022.01" #"am5f7b12r1"
+stem: "am5/2022.01" # stem=projectID/version
+
+# Experiments
+exp:
+  - expname: "c96L65_am5f1a0r0_amip"
+    platform: "gfdl.ncrc5-intel22-classic"
+    target: "prod-openmp"
+    ppyaml: "pp_c96L65.yaml"
```

Comment:
I like this, mostly. So one does a

fre pp wrapper -y am5-yamls.yaml -p gfdl.ncrc5-intel22-classic -t prod-openmp -e c96L65_am5f1a0r0_amip

and first the am5-yamls.yaml is read, and then the pp_c96L65.yaml is also read. Yes?

I like this as is, but have some reservations about `target` and `platform`. For postprocessing, the target and platform are mostly used for directories, while the experiment is truly specific to the pp_c96L65.yaml settings.

Since we're passing the platform, target, and experiment to the `fre` tools, I think it's unnecessarily redundant to store them in the yaml also.

Reply:
Yes! The `am5-yamls.yaml` would be passed with the `-y` option in the command, as well as the `-e ...` option. So then, if the experiment passed as a click option matches the expname listed in the am5-yamls, it uses the information associated with that experiment name. From your example, 
`fre pp wrapper -y am5-yamls.yaml -p gfdl.ncrc5-intel22-classic -t prod-openmp -e c96L65_am5f1a0r0_amip`

1. `am5-yamls.yaml` is read
2. `-e c96L65_am5f1a0r0_amip` is given so it looks through the expname section of the yaml to match the experiment name given
3. uses the info associated with c96L65_am5f1a0r0_amip, being that specific experiment's ppyaml

I completely agree on the target and platform Chris! I thought about it more this morning, completely redundant and unnecessary.